app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,19 +27,44 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * 모든 프로덕트를 반환합니다.
+     *
+     * @return 모든 프로덕트가 담긴 리스트
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 특정한 프로덕트를 가져옵니다.
```

Comment:
`가져온다`는 표현은 모호합니다. 가져와서 어딘가에 저장한다는 것인지, 메소드 호출 결과로 리턴한다는 것인지 분명하지 않아요. 메소드의 역할을 리턴값 위주로 설명해 보세요.
```suggestion
     * 프로덕트를 리턴합니다.
```